.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# based on wild guessing, might not work first try ngl
+
+name: Release
+
+on:
+  push:
+    branches:
+      - master # this tells it to run when commits are made to master
+
+jobs:
+  check_versions:
+    runs-on: ubuntu-latest
+    outputs: # this is kinda fucky in the way it works ngl
+      version: ${{ steps.check_versions.outputs.version }} # this uses the id from the step itself, not the job (they are the same)
+      continue: ${{ steps.check_versions.outputs.continue }}
+    steps:
+      - uses: actions/checkout@v2
+        id: check_versions
+      - run: git describe --tags --abbrev=0 > /tmp/latest_tag # we get the latest tag and store it
+      - run: jq -r ".version" package.json > /tmp/current_version # we get the current package.json version and store it
```

Comment:
i think the ubuntu image for github actions comes with `jq` preinstalled but i'm not entirely sure